static/css/components/read-panel.less

```diff
@@ -71,6 +71,7 @@
 
 .cta-section-title.world-cat-link{
   margin-top:0;
+  display: inline;
```

Comment:
The fact we're modifying 2 components sends a signal that this is organized incorrectly. Changing one component should not impact other components.

We should use a different class explaining the context of the new link e.g. edition__world-cat-link.

A change here should be unnecessary.

Reply:
I replaced `<p>` with `<span>` so I do we `display: inline;`

Reply:
So according to read-panel.less it is this component:
https://github.com/internetarchive/openlibrary/wiki/Design-Pattern-Library#read-panel

You however are touching https://github.com/internetarchive/openlibrary/wiki/Design-Pattern-Library#editionstable (/css/components/editions.less)

Use the new class to remove the overlap between these comments and drop this style change.

Reply:
I have removed all changes from read.panel.less in the previous commit.